packages/amplify-graphql-relational-transformer/src/graphql-belongs-to-transformer.ts

```diff
@@ -185,7 +208,7 @@ const validate = (config: BelongsToDirectiveConfiguration, ctx: TransformerConte
     });
   });
 
-  if (!isBiRelation) {
+  if (!isBiRelation && dbType === DDB_DB_TYPE) {
     throw new InvalidDirectiveError(
       `${config.relatedType.name.value} must have a relationship with ${object.name.value} in order to use @${directiveName}.`,
```

Comment:
wouldn't we have to check this for RDS DB types as well?

Reply:
Not required. RDS doesn't require bi directional relationship. Single field defines the relationship on both ends, so customer can use belongsTo without hasMany or hasOne.